demonstrations/tutorial_unitary_designs.py

```diff
@@ -122,223 +128,266 @@
-group. With just a subset of the full unitary group, we can evaluate complex
-expressions that would be otherwise intractable.
-
-A surprising result about unitary designs is that they exist for all possible
-combinations of :math:`t` and :math:`d` [#Roy]_. There are some known lower
-bounds for the number of unitaries required; for example, a 2-design in
-dimension has minimum size :math:`d` is :math:`d^4 - 2d^2 + 2` [#Gross]_,
-[#Roy]_.  However, actually finding the sets (and in particular, finding ones
-with minimal size), is a challenging problem [#Bannai]_, though very recently
-some constructions have been put forward [#Nakata]_.
-
-
-.. admonition:: Fun fact
-
-    Applying the elements of a unitary design to a fixed pure state produces
-    a set of vectors that form a complex projective design [#DankertThesis]_!
-
-Unitary :math:`t`-designs in action
------------------------------------
-
-Unitary designs come into play in applications that require randomization, or
-sampling of random unitaries--essentially, they can be used as a stand-in for
-the Haar measure. The way in which the unitaries are used in the application may
-place restrictions on the value of :math:`t` that is required; arguably the most
-common is the unitary 2-design.
-
-While in general unitary designs are hard to construct, there are well known
-results for unitary 1-, 2-, and 3-designs based on familiar objects in quantum
-computing. Before we see what those are, let's explore why such designs are
-useful.
-
-Average fidelity
-^^^^^^^^^^^^^^^^
-A key application of unitary 2-designs is benchmarking quantum
-operations. Suppose we have a noisy quantum channel :math:`\Lambda` that should
-perform something close to the unitary operation :math:`V`.  What can we say
-about the performance of this channel?
-
-One metric of interest is the *fidelity*. Consider the state :math:`|0\rangle`.
-In an ideal case, applying :math:`V` gives :math:`V|0\rangle`.  But applying the
-channel :math:`\Lambda` gives us something a little different. Since it's noisy,
-we must consider the state as a density matrix. The action of :math:`\Lambda` on
-our starting state is :math:`\Lambda(|0\rangle \langle 0|)`.  If :math:`\Lambda`
-was perfect, then :math:`\Lambda(|0\rangle \langle 0|) = V|0\rangle \langle
-0|V^\dagger`, and the fidelity is
+configurations become increasingly sphere-like. Looking at this in a different
+way, the more complex a function becomes as its degree increases, the closer the
+:math:`t`-design must be to a sphere; we need to evaluate the function at more
+points in order to gain sufficient information when a function is varying more
+quickly due to a higher degree. In 3 dimensions, a function with degree 2 can
+be approximated with something as simple as a tetrahedron, despite the fact that
+it doesn't look spherical at all, while more complex functions require more
+points and thus more complex configurations for the design.  Spherical designs
```

Comment:
```suggestion
points and thus more intricate configurations for the design.  Spherical designs
```
or other synonym to avoid repeated use of complex